src/com/boxboat/jenkins/pipeline/build/BoxBuild.groovy

```diff
@@ -59,7 +59,7 @@ class BoxBuild extends BoxBase<BuildConfig> implements Serializable {
         Compose.down(config.composeProfileMap.get(profile), profile)
     }
 
-    def push() {
+    def push(List<String> additionalImageTags) {
```

Comment:
you can skip the overload and just use `List<String> additionalImageTags = []`, it compiles down to the same thing as the overload and is a little more clear